### DISCUSSION TITLE
No token found in response.

### DISCUSSION THREAD

**Discussion by nickpieters** [pypolestar/polestar_api#196] [Q&A] [answered]
Hi

Just got my new PS3. When i use the login/password i get the No Token found in response. In the description of the polestar_api i see a screenshot where the VIN is showed, however i only get the username/password combination. But when entering the username/password combination it just doesn't work..

Am i missing something?

Thanks.

**Comment by jschlyter** [comment, marked as answer]
No, you are not missing anything - Polestar is, see #197.
